git-remote-ipfs/src/Network/IPFS/Git/RemoteHelper/Client.hs

```diff
@@ -264,8 +262,9 @@ updateRemoteUrl root = do
         IpfsPathIpfs _    -> viaConfig (remoteUrlScheme url) root
   where
     viaIpns name = do
-        logInfo "Updating IPNS"
         let ipnsTarget = "/ipfs/" <> cidToText root
+        logInfo $
+            fmt ("Updating IPNS link " % ftxt % " to " % ftxt) name ipnsTarget
```

Comment:
I might have missed the `ftxt` combinator -- genuine curiosity, why did you prefer it over `stext`?

Reply:
One character less 🤣 

I have this little helper module `Network.IPFS.Git.RemoteHelper.Format`...